flask_resty/sorting.py

```diff
@@ -115,7 +115,7 @@ def get_criteria(self, view, field_orderings):
     def get_criterion(self, view, field_ordering):
         field_name, asc = field_ordering
         column = self.get_column(view, field_name)
-        return column if asc else column.desc()
+        return sa.nullslast(column) if asc else sa.nullsfirst(column.desc())
```

Comment:
is the explicit nulls * to be safe and consistent or does this change something from the default?

Reply:
Different  databases have different behavior related to how NULLs are sorted (for example sqlite (used for tests in flask-resty) and postgres handle nulls order by in different way). So we need to make it consistent to avoid issues reletad to this.